diff --git a/node_modules/@ckeditor/ckeditor5-engine/src/view/renderer.js b/node_modules/@ckeditor/ckeditor5-engine/src/view/renderer.js
index a2a1a14..a7942b4 100644
--- a/node_modules/@ckeditor/ckeditor5-engine/src/view/renderer.js
+++ b/node_modules/@ckeditor/ckeditor5-engine/src/view/renderer.js
@@ -633,7 +633,9 @@ export default class Renderer extends ObservableMixin() {
                 // @if CK_DEBUG_TYPING // 		'color: green;font-weight: bold', ''
                 // @if CK_DEBUG_TYPING // 	);
                 // @if CK_DEBUG_TYPING // }
-                updateTextNode(actualDomChildren[i], expectedDomChildren[i].data);
+                if (!this.isComposing) {
+                    updateTextNode(actualDomChildren[i], expectedDomChildren[i].data);
+                }
                 i++;
                 // @if CK_DEBUG_TYPING // if ( window.logCKETyping ) {
                 // @if CK_DEBUG_TYPING // 	console.groupEnd();
